membrane_curvature/curvature.py

```diff
@@ -46,7 +46,7 @@
 import numpy as np
 
 
-def gaussian_curvature(Z):
+def gaussian_curvature(Z, *varargs):
```

Comment:
I am wondering why this is `*varargs` and not `dy` `dx`. 
Since there are no tests for `*varargs`, if we keep it maybe we should add some non-uniform tests. Otherwise, maybe it would be better to have `gaussian_curvature(Z, dx=1, dy=1)`?

Reply:
I followed the prototype from [numpy.gradient](https://numpy.org/doc/stable/reference/generated/numpy.gradient.html). I think having keyword args dx, and dy are a good idea. I am however opposed to setting a default dx, dy value to be 1. I don't think that's a sensible default, most datasets won't be sampled at 1 Angstrom spacing, and folks can make mistakes by accident. WDYT?

Non-uniform spacing tests are also a good idea. I don't have time now to add that to this PR now but can commit to it in a week or two. I do know that some folks are using this library and I think it would be good to merge this PR sooner rather than later. I hope that the test won't delay that.